[harfbuzz] Unable to build

```
ImportError: No module named six
make[2]: *** [Makefile:859: install-data-local] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/harfbuzz-1.7.6.tar.bz2.dir/harfbuzz-1.7.6/docs'
make[1]: *** [Makefile:602: install-am] Error 2
make[1]: Leaving directory '/usr/local/tmp/crew/harfbuzz-1.7.6.tar.bz2.dir/harfbuzz-1.7.6/docs'
make: *** [Makefile:514: install-recursive] Error 1
harfbuzz failed to build: `make DESTDIR=/usr/local/tmp/crew/dest install` exited with 2
```
Missing pip module?

--- a/packages/freetype.rb
+++ b/packages/freetype.rb
@@ -7,19 +7,31 @@ class Freetype < Package
   source_url 'http://download.savannah.gnu.org/releases/freetype/freetype-2.9.tar.bz2'
   source_sha256 'e6ffba3c8cef93f557d1f767d7bc3dee860ac7a3aaff588a521e081bc36f4c8a'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/freetype-2.9-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/freetype-2.9-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/freetype-2.9-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/freetype-2.9-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '7fd752cc084ee8a101030e682b0dbce39551049fddbf547762b6f8e60450e131',
+     armv7l: '7fd752cc084ee8a101030e682b0dbce39551049fddbf547762b6f8e60450e131',
+       i686: '1c2688c3070e37d61db309a6310a5a613aa89c419a89cbc79d9055b56c1c82fc',
+     x86_64: '1490d08a9b385ef3abc323c914aaf218f0b59961885e3cda64278d971ace71c9',
+  })
 
   depends_on 'expat'
   depends_on 'libpng'   # freetype needs zlib optionally. zlib is also the dependency of libpng
   depends_on 'bz2'
   depends_on 'harfbuzz'
-  
+
   def self.build
-      system "./configure \
-              CFLAGS=\" -fPIC\" \
-              --prefix=#{CREW_PREFIX} \
-              --libdir=#{CREW_LIB_PREFIX} \
-              --with-harfbuzz"
-      system "make"
+    system "./configure \
+            CFLAGS=\" -fPIC\" \
+            --prefix=#{CREW_PREFIX} \
+            --libdir=#{CREW_LIB_PREFIX} \
+            --with-harfbuzz"
+    system "make"
   end
 
   def self.install

--- a/packages/freetype_sub.rb
+++ b/packages/freetype_sub.rb
@@ -7,6 +7,18 @@ class Freetype_sub < Package
   source_url 'http://download.savannah.gnu.org/releases/freetype/freetype-2.9.tar.gz'
   source_sha256 'bf380e4d7c4f3b5b1c1a7b2bf3abb967bda5e9ab480d0df656e0e08c5019c5e6'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/freetype_sub-2.9-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/freetype_sub-2.9-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/freetype_sub-2.9-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/freetype_sub-2.9-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'f180b27eb703e0e7680f53ee4396a7e9cfb4093d9f29b7d16fffad6c6476c912',
+     armv7l: 'f180b27eb703e0e7680f53ee4396a7e9cfb4093d9f29b7d16fffad6c6476c912',
+       i686: '71fb1bace0552659cfc86ef6ed88bbb1ce5c618061107381618f1a1060c47397',
+     x86_64: '3ccd3b201456c5d9f2dac55311ed3f337edd230a894fae5e821072515a3d5d5f',
+  })
 
   depends_on 'expat'
   depends_on 'libpng'   # freetype needs zlib optionally. zlib is also the dependency of libpng
@@ -28,5 +40,4 @@ class Freetype_sub < Package
   def self.postinstall
     system "find #{CREW_BREW_DIR}/* -name freetype*.tar |xargs rm -rf"  # make sure to delete downloaded files
   end
-  
 end

--- a/packages/fskit.rb
+++ b/packages/fskit.rb
@@ -7,18 +7,29 @@ class Fskit < Package
   source_url 'https://github.com/jcnelson/fskit/archive/dd9a8dc844b81afc63e5067397786a32ecb66008.tar.gz'
   source_sha256 'fe682890ebab9226d65fc6ebfb8b3619c0d5a93e3161787cea9d01ad23d3a83a'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/fskit-dd9a8-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/fskit-dd9a8-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/fskit-dd9a8-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/fskit-dd9a8-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'b51b018c892a80340f4957b64df87ea55f1a5dbe7bd5dc574bd504b8cefc3429',
+     armv7l: 'b51b018c892a80340f4957b64df87ea55f1a5dbe7bd5dc574bd504b8cefc3429',
+       i686: '7463ea0b68a5b29ea3b4162ccade4556d380ed55ba6c391de74c9821b8156388',
+     x86_64: '7ed7cade6d97d596a894b024785d910fa378d11017fb83487ab8a43ac446a784',
+  })
 
   depends_on 'attr'
 
-    
   def self.patch
     # for x86_64 architecture, build library should be modified to #{CREW_PREFIX}/lib64
     if ARCH == "x86_64"
-       puts "Patch build file for x86_64 ..." 
-        `sed -i -- 's/lib/lib64/g' buildconf.mk` # replace lib by lib64
+      puts "Patch build file for x86_64 ..."
+      `sed -i -- 's/lib/lib64/g' buildconf.mk` # replace lib by lib64
     end
   end
-    
+
   def self.build
     system "make"
   end

--- a/packages/harfbuzz.rb
+++ b/packages/harfbuzz.rb
@@ -7,6 +7,18 @@ class Harfbuzz < Package
   source_url 'https://github.com/harfbuzz/harfbuzz/releases/download/1.7.6/harfbuzz-1.7.6.tar.bz2'
   source_sha256 'da7bed39134826cd51e57c29f1dfbe342ccedb4f4773b1c951ff05ff383e2e9b'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/harfbuzz-1.7.6-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/harfbuzz-1.7.6-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/harfbuzz-1.7.6-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/harfbuzz-1.7.6-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'd92567e640f2b847ff35d1dcb5e0fcabaa640cea8c709755a3fed8b1ed59a598',
+     armv7l: 'd92567e640f2b847ff35d1dcb5e0fcabaa640cea8c709755a3fed8b1ed59a598',
+       i686: '220ae1416d6fb21e9d5621d97c66e65dddd15eb63108fb4c5e3d5fe6c75e662a',
+     x86_64: '3dd360778d0ffbd12b23a9d1e2fe5d3031f8a93eb9f9618cd430dc91349bee7d',
+  })
 
   depends_on 'glib'
   depends_on 'icu4c'
@@ -18,6 +30,8 @@ class Harfbuzz < Package
   end
 
   def self.install
+    system "pip install six"
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "pip uninstall --yes six"
   end
 end

--- a/packages/heroku.rb
+++ b/packages/heroku.rb
@@ -6,7 +6,20 @@ class Heroku < Package
   version '6.15.31'
   source_url 'https://github.com/heroku/cli/archive/v6.15.31.tar.gz'
   source_sha256 '940c8083b90f95c5400df37dd0de19e423bd665e729c3d8e03b4c1ed34852d95'
-  
+
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/heroku-6.15.31-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/heroku-6.15.31-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/heroku-6.15.31-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/heroku-6.15.31-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '14bedf57aba0005bc6c8446dc387b127a18de0fc54e2c22184e1127380447937',
+     armv7l: '14bedf57aba0005bc6c8446dc387b127a18de0fc54e2c22184e1127380447937',
+       i686: '68024018ccd7fa5053d07a32694a3040c7f94cd0422d27545992391f71a7e71c',
+     x86_64: '1cbc40d073dfb2ef09b36ce258ed581e155f2d29634ed8080c316b6b3177eb62',
+  })
+
   depends_on 'yarn'
 
   def self.build

